README.md

```diff
@@ -104,47 +104,91 @@ $ gem install fortnox-api
+
+Start your journey at [Fortnox getting started guide](https://developer.fortnox.se/getting-started/). Note that there's a script to authorize the Fortnox app to your Fortnox account bundled with this gem to help you getting started, see [Initialization](#initialization). Also read [Authorizing your integration](https://developer.fortnox.se/general/authentication/).
+
+Things you need:
+- A Fortnox developer account
+- A Fortnox app with:
+  - Service account setting enabled (it's used in server to server integrations, which this is)
+  - Correct scopes set
+  - A redirect URL (just use a dummy URL if you want to, you just need the parameters send to that URL)
+- A Fortnox test environment so that you can test your integration.
+
+When you have authorized your integration you get an access token. It's a JWT with an expiration time of 1 hour. You also get a refresh token that lasts for **31 days**. When a new access token is requested, a new refresh token is also provided and the old one is invalidated. As long as the refresh token is valid, the gem will do all of this automatically. *You just need to make sure the gem makes a request the Fortnox API before the current refresh token expires*, otherwise you need to start over again with the [Initialization](#initialization).
+
+## Initialization
+As described in the [Authorization](#authorization) section, the gem refreshes the tokens automatically, but you need to initialize the process. There a script provided in `bin/get_tokens` to get your initial tokens.
+
+### Configuration
+The gem is configured in a `configure` block.
+
+Due to Fortnox use of refresh tokens, the gem needs a storage of some sort to keep the tokens. The only thing the storage needs to expose is `access_token` and `refresh_token`. A very simplistic storage would look like this:
 
 ```ruby
-Fortnox::API.configure do |config|
-  config.client_secret = 'P5K5wE3Kun'
-  config.access_token = '3f08d038-f380-4893-94a0a0-8f6e60e67a'
+class MyStorage
+  attr_accessor :access_token
+  attr_accessor :refresh_token
+
+  def initialize(access_token, refresh_token)
+    @access_token = access_token
+    @refresh_token = refresh_token
+  end
 end
 ```
-Before you start using the gem.
 
-### Multiple AccessTokens
+And could then be used like this:
 
-Fortnox uses quite low [API rate limits](https://developer.fortnox.se/blog/important-implementation-of-rate-limits/). The limit is for each access token, and according to Fortnox you can use as many tokens as you like to get around this problem. This gem supports handeling multiple access tokens natively. Just set the `access_tokens` (in plural, compared to `access_token` that only takes a String) to a list of strings:
 ```ruby
 Fortnox::API.configure do |config|
-  config.client_secret = 'P5K5wE3Kun'
-  config.access_tokens ['a78d35hc-j5b1-ga1b-a1h6-h72n74fj5327', 's2b45f67-dh5d-3g5s-2dj5-dku6gn26sh62']
+  config.storage = MyStorage.new(
```

Comment:
Wouldn't this be:
```
Fortnox::API.configure do |config|
  config.storage = MyStorage.new
end
```
I would expect the implementation of the storage to handle any loading from env on init etc ...